modules/builders/package.json

```
@@ -32,6 +32,7 @@
     "@angular-devkit/core": "DEVKIT_CORE_VERSION",
     "@nguniversal/common": "0.0.0-PLACEHOLDER",
     "browser-sync": "^2.26.7",
+    "express": "EXPRESS_VERSION",
```

In here, you can use the actual version, no need to use the replacements. 
```suggestion
    "express": "^4.17.1",
```